lara-typescript/src/interactive-api-client/README.md

```diff
@@ -0,0 +1,37 @@
+# LARA Interactive API
+
+### This documentation is meant to be used by LARA interactive developers.
+
+#### Setup and webpack configuration
+
+The API is available under `window.LARA.InteractiveAPI` object / namespace once the plugin is initialized by LARA.
```

Comment:
This sounds like it should be updated. In the client I'd assume this object is available immediately. In other words it shouldn't have to wait until LARA "initializes the interactive".